src/core_plugins/kibana/public/dev_tools/hacks/disable.js

```diff
@@ -0,0 +1,16 @@
+import chrome from 'ui/chrome';
+import modules from 'ui/modules';
+
+modules.get('kibana').run(function ($rootScope, $location) {
```

Comment:
This was removed in favor of disabling console and hiding the dev tools application when there aren't any tools registered.